$NetBSD: patch-al,v 1.1 2001/03/27 11:53:18 agc Exp $

Change the manual page section, so that it doesn't conflict
with the digest(1) manpage.

--- Doc/man/digest.1	2001/03/27 11:46:32	1.1
+++ Doc/man/digest.1	2001/03/27 11:46:42
@@ -1,4 +1,4 @@
-.TH digest 1
+.TH digest 8
 .SH NAME
 digest \- receive a file for a digest, or create and mail a digest
 .LP
